internal/kafka/internal/handlers/validation.go

```diff
@@ -260,8 +260,9 @@ func ValidateKafkaUpdateFields(kafkaUpdateRequest *private.KafkaUpdateRequest) h
 			stringSet(&kafkaUpdateRequest.KafkaVersion) ||
 			stringSet(&kafkaUpdateRequest.KafkaIbpVersion) ||
 			stringSet(&kafkaUpdateRequest.DeprecatedKafkaStorageSize) ||
-			stringSet(&kafkaUpdateRequest.MaxDataRetentionSize)) {
-			return errors.FieldValidationError("Failed to update Kafka Request. Expecting at least one of the following fields: strimzi_version, kafka_version, kafka_ibp_version, kafka_storage_size or max_data_retention_size to be provided")
+			stringSet(&kafkaUpdateRequest.MaxDataRetentionSize) ||
+			arrays.IsNotNilPredicate(kafkaUpdateRequest.Suspended)) {
+			return errors.FieldValidationError("Failed to update Kafka Request. Expecting at least one of the following fields: strimzi_version, kafka_version, kafka_ibp_version, kafka_storage_size, max_data_retention_size or suspended to be provided")
```

Comment:
Error messages should start with lowercase

Reply:
@ziccardi - there are a lot of ocurrences of errors starting with uppercase across kfm. I will change this one and create a jira to tackle others

Reply:
@pawelpaszki @ziccardi ,

A possible option is to add the pattern in the `forbididgo` configuration of golangci-lint. For example, adding fmt.Errorf starting with a literal string that begins with uppercase inside the first parameter of it. We could also add GeneralError with the same pattern. Same for errors.Errorf and other cases where we know we can reliably perform the check without false positives.

This won't detect all scenarios nor warn us of all of them but it would be an improvement.

Reply:
A potentially better alternative, or one to be used in conjunction with my previous proposal is to use the `revive` linter with the `error-strings`  rule `https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#error-strings` although we should explore in the doc / behavior how far it hows and the chance of false positives.

Of course all of this means that it will start to detect the errors so those should be fixed together with the corresponding linters changes